src/js/hora.js

```diff
@@ -335,196 +383,11 @@ define([
             }
         };
 
-        Hora.navigation = {
```

Comment:
@Helen-Mobify Not that I know of, but you could make your changes and then add a link to an old revision. There's a list of revisions (for homepage) here: https://github.com/mobify/hora.js/wiki/Home/_history

Click on the SHA: https://github.com/mobify/hora.js/wiki/Home/beaccd754174b7bdaea27f1da19e45411d751138